sql/sql_show.cc

```diff
@@ -8925,7 +8926,7 @@ static my_bool run_hton_fill_schema_table(THD *thd, plugin_ref plugin,
   struct run_hton_fill_schema_table_args *args=
     (run_hton_fill_schema_table_args *) arg;
   handlerton *hton= plugin_hton(plugin);
-  if (hton->fill_is_table && hton->state == SHOW_OPTION_YES)
+  if (hton->fill_is_table && ha_storage_engine_is_enabled(hton))
```

Comment:
Looks like dead code. No hton exporting `fill_is_table`. Not exported by `binlog` either. So just remove `hton->state` check.